pkg/controller/dataimportcron-conditions.go

```diff
@@ -87,3 +87,16 @@ func updateConditionState(condition *cdiv1.ConditionState, status corev1.Conditi
 		condition.Message = message
 	}
 }
+
+func updateDataImportCronFailingMetric(cron *cdiv1.DataImportCron, increment bool) {
```

Comment:
`updateDataImportCronOutdatedMetric` might be a better name?

Reply:
`updateDataImportCronOutdatedMetric()` is the better choice, thx
Prefer to keep the straightforward `increment` word so there is no place for misunderstanding about whats going to happen